database.py

```diff
@@ -0,0 +1,37 @@
+from flask_pymongo import PyMongo
+
+class Database():
+    def __init__(self, app):
+        self.mongo = PyMongo(app)
+
+    def set_username(self, id, username):
+        self.mongo.db.user.insert_One({"username": username})
+
+    def get_username(self, id, username):
+        return self.mongo.db.user.findOne({"username": username})
+
+    def set_password(self, id, password):
+        self.mongo.db.user.insert_One({"password": password})
```

Comment:
Shouldn't this be?: 

```
db.user.update({"id": id},  
    {
        $set:  { 
            "password" : password
        }
    }
)

```

Since this function is trying to update an existing user but not completely replace the rest of the user info? I don't know how pymongo will handle the $set operator though.